dali/test/python/test_utils.py

```diff
@@ -186,6 +186,33 @@ def get_absdiff(left, right):
         _check_absdiff()
     return _get_absdiff(left, right)
 
+def dump_as_core_artifacts(image_info, lhs, rhs, iter=None, sample_idx=None):
+    import_numpy()
+    import_pil()
+    from pathlib import Path
+    path = "/opt/dali" if os.path.exists("/opt/dali") else os.getcwd()
```

Comment:
That's not good. A regular user who happens to have DALI there won't be able to write to `/opt`.

Reply:
added a check for write permissions